apps/ecs/index.js

```diff
@@ -52,8 +52,41 @@ module.exports = {
     },
     '/digital-right-to-work-service': {
       fields: ['use-digital-right-to-work'],
+      forks: [
+        {
+          target: '/eu-settlement-scheme',
+          condition: {
+            field: 'use-digital-right-to-work',
+            value: 'no'
+          }
+        }
+      ],
+      next: '/right-to-work-check'
+    },
+    '/eu-settlement-scheme': {
+      fields: ['worker-applied-eu-settlement-scheme'],
+      forks: [
+        {
+          target: '/arc-card',
+          condition: {
+            field: 'worker-applied-eu-settlement-scheme',
+            value: 'none-of-above'
+          }
+        }
+      ],
+      next: '/right-to-work-check'
+    },
+    '/right-to-work-check': {
```

Comment:
Pathname should be `/request-check`, its been mentioned in comment in the figma for this screen.